src/utils/auth.js

```diff
@@ -0,0 +1,11 @@
+const tokenKey = 'token';
+
+const saveUserToken = token => window.localStorage.setItem(tokenKey, token);
+const getUserToken = () => window.localStorage.getItem(tokenKey);
+const removeUserToken = () => window.localStorage.removeItem(tokenKey);
+
+export default {
+  saveUserToken,
+  getUserToken,
+  removeUserToken,
+};
```

Comment:
Good initiative here @codeshifu ! You've taken care of the token logic in one file for reusability purpose.
Hopefully we would update this file with a `decode token` method when the need arise.